packages/cardpay-reward-programs/cardpay_reward_programs/main.py

```diff
@@ -20,12 +20,12 @@
 )
 cached_client.set_as_default_client()
 
-for expected_env in ["ENVIRONMENT"]:
-    if expected_env not in os.environ:
-        raise ValueError(f"Missing environment variable {expected_env}")
 
 SENTRY_DSN = os.environ.get("SENTRY_DSN")
 if SENTRY_DSN is not None:
+    for expected_env in ["ENVIRONMENT"]:
+        if expected_env not in os.environ:
+            raise ValueError(f"Missing environment variable {expected_env}")
```

Comment:
Note - there may be a conflict here, but the change matches what we'll have done elsewhere.